chaosinstana/api.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from typing import Any, Dict
+
+from chaoslib.types import Configuration, Secrets
+from logzero import logger
+import requests
+
+
+__all__ = ['call_api', 'get_all_events']
+
+
+def call_api(path: str, configuration: Configuration,
+             secrets: Secrets, query: Dict[str, Any] = None,
+             body: Dict[str, Any] = None, method: str = "GET",
+             headers: Dict[str, str] = None) -> requests.Response:
+    """
+    TBD
+    """
+    # location given here
+    URL = "https://httpbin.org/get"
+
+    # defining a params dict for the parameters to be sent to the API
+    # PARAMS = {'address':location}
+
+    # sending get request and saving the response as response object
+    r = requests.get(url=URL)
+    # extracting data in json format
+    data = r.json()
+
+    return data
+
+
+def get_all_events(from_time: str, to_time: str,
+                   configuration: Configuration, secrets: Secrets) -> str:
+    """
+     get all events from instana within a time window, given by the from_time
+     and the to_time for details of the api see
+     https://instana.github.io/openapi/#tag/Events
+    """
+    instana_host = configuration.get("instana_host")
+    instana_api_token = secrets.get("instana_api_token")
+
+    if (instana_host and instana_api_token):
```

Comment:
parenthesis are seldom needed in Python. This is a case where you can drop them :)